.github/workflows/swift-toolchain.yml

```diff
@@ -707,28 +707,25 @@ jobs:
             ${{ steps.export-binary-paths.outputs.swift_compatibility_symbols }}
 
   early_swift_driver:
-    # TODO: Build this on Windows too.
-    if: inputs.build_os == 'Darwin'
-    needs: [sqlite]
     runs-on: ${{ inputs.default_build_runner }}
 
     strategy:
       fail-fast: false
-      matrix: ${{ fromJSON(inputs.host_matrix) }}
 
-    name: ${{ matrix.os }} ${{ matrix.arch }} Early Swift Driver
```

Comment:
This already builds on macOS. Could you change the matrix to `build_matrix` if this is what we need?